Sources/TuistGenerator/Mappers/AutogeneratedSchemesProjectMapper.swift

```diff
@@ -4,9 +4,13 @@ import TuistCore
 /// A project mapper that auto-generates schemes for each of the targets of the `Project`
 /// if the user hasn't already defined schemes for those.
 public final class AutogeneratedSchemesProjectMapper: ProjectMapping {
+    private let enableCodeCoverage: Bool
+
     // MARK: - Init
 
-    public init() {}
+    public init(_ enableCodeCoverage: Bool) {
```

Comment:
minor: in this case perhaps keeping the named parameter will help clarify what the boolean does at the point of use 

e.g. `AutogeneratedSchemesProjectMapper(enableCodeCoverage: true)` vs `AutogeneratedSchemesProjectMapper(true)`

Reply:
tbh `FooBar(true)` did give me a little bit of a weird vibe. I'll take care of this with the other tidying up. Thanks again!